resources/views/task/form.blade.php

```diff
@@ -0,0 +1,53 @@
+@extends('layouts.main')
+
+@section('content')
+    <div class="container">
+        @if($task->exists)
+            <form action="{{ route('task.update', $task) }}" method="post">
+                {{ method_field('PUT') }}
+            @else
+
+            <form action="{{ route('task.store') }}" method="post">
+
+            @endif
+
+                {{ csrf_field() }}
+                <div class="mdl-grid">
+                    <div class="mdl-cell mdl-cell--6-col mdl-cell--3-offset">
```

Comment:
Bad indentation. Remove the empty lines on form validation and indent all lines.